.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -19,3 +19,23 @@ A clear and concise description of what you want to happen.
 
 **Describe alternatives you've considered**
 A clear and concise description of any alternative solutions or features you've considered.
+
+----------------
+For Change Control Board: Issue Review
+----------------
+This review should occur before any development is performed as a response to this issue.
+- [ ] 1. Is it tagged with a type: defect or task?
```

Comment:
This is a feature request template, should only task be mentioned, and should item 4 be modified?